### PR TITLE
Create `StorageWhiteboxVerification` test class in the src folder

### PR DESCRIPTION
**Pull request by dgarus**
part of #445 

- removed 'StorageWrapper' class
- common tests are joined to `StorageWhiteboxVerification`
- created separate test verification classes for every storage implementation

### CODE REVIEW

File: asto-core/src/test/java/com/artipie/asto/VertxFileStorageVerificationTest.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        return new VertxFileStorage(
+            this.temp.resolve("base"),
+            this.vertx
+        );
+    }
+
+    @Override
+    protected Optional<Storage> newBaseForRootSubStorage() {
+        // @checkstyle MethodBodyCommentsCheck (5 lines)
+        //todo #445 fix VertxFileStorage verification tests for case root sub storage.
```
Comment (olenagerasimova) [inline asto-core/src/test/java/com/artipie/asto/VertxFileStorageVerificationTest.java:63]: please, check and correct todo formatting https://www.yegor256.com/2010/03/04/pdd.html

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@dgarus thanks, looks good! Just correct todo formatting please

File: asto-core/src/test/java/com/artipie/asto/VertxFileStorageVerificationTest.java (hunk 1 above)
Comment (dgarus) [reply asto-core/src/test/java/com/artipie/asto/VertxFileStorageVerificationTest.java]: I decided to remove extra comments.

**Comment by dgarus** [comment]
@olenagerasimova thanks for the review!
I fixed your comments.

**Review by olenagerasimova — APPROVED** [review]
@dgarus thanks

**Pull request merged**